buildpipeline/DotNet-CoreClr-Trusted-Mac.json

```diff
@@ -292,7 +292,9 @@
       "value": "$(Build.BuildNumber)"
     }
   },
-  "demands": [],
+  "demands": [
```

Comment:
Is there only 1 agent in the pool that works?

Reply:
Yes